assets/styles/main.css

```diff
@@ -0,0 +1,175 @@
+
+
+/********************************************************
+help borders used at development
+*********************************************************/
+
+/*header {
+  border: 4px solid blue;
+}
+
+main {
+  border: 4px solid green;
+}
+
+section {
+  border: 2px solid red;
+}
+
+article {
+  border: 1px solid red;
+}
+
+div {
+  border: 1px dashed red;
+}
+
+footer {
+  border: 4px solid blue;
+}*/
+
+
+/********************************************************
+header
+*********************************************************/
+
+header {
+  position: fixed;
+  z-index: 100;
+  width: 100%;
+  top: 0px;
+   /* Marlin */
+  background-color: rgba(82,90,136, 1);
+  /* Gold */
+  color: rgba(200,179,116, 1);
+}
+
+header div {
+  display: inline-block;
+  margin-left: 50px;
+}
+
+header nav {
+  display: inline-block;
+  float: right;
+}
+
+header ul {
+  margin-top: 20px;
+}
+
+header li {
+  display: inline-block;
+}
+
+header a {
+  text-decoration: none;
+  /* Gold */
+  color: rgba(200,179,116, 1);
+  margin-right: 50px;
+}
+
+
+/********************************************************
+main for INDEX, PORTFOLIO, BLOG
+*********************************************************/
+main {
+  margin-top: 58px;
+}
+
+.hero {
+  position: relative;
+  clear: both;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  background-position: center;
+  text-align: center;
+  vertical-align: middle;
+  height: 300px;
+  line-height: 256px;
```

Comment:
Tuck away for future... Hard-coded pixel sizes can get tricky if you're trying to make a site responsive. But, for now, on desktop, it's lovely.